src/blender/scripts/colors.py

```diff
@@ -123,32 +123,30 @@ def load_font(font_path):
 material_object1.diffuse_color = params["diffuse_color"]
 material_object1.specular_color = params["specular_color"]
 material_object1.specular_intensity = params["specular_intensity"]
-material_object1.alpha = params["alpha"]
 
 # TITLE 2 - Change the material settings (color, alpha, etc...)
 material_object2 = bpy.data.materials["Title2.Material"]
 material_object2.diffuse_color = params["diffuse_color"]
 material_object2.specular_color = params["specular_color"]
 material_object2.specular_intensity = params["specular_intensity"]
-material_object2.alpha = params["alpha"]
 
 # TITLE 3 - Change the material settings (color, alpha, etc...)
 material_object3 = bpy.data.materials["Title3.Material"]
 material_object3.diffuse_color = params["diffuse_color"]
 material_object3.specular_color = params["specular_color"]
 material_object3.specular_intensity = params["specular_intensity"]
-material_object3.alpha = params["alpha"]
 
 # BACKGROUND - Change the material settings (color, alpha, etc...)
 material_object4 = bpy.data.materials["Background.Material"]
 material_object4.specular_color = params["specular_color_bg"]
 material_object4.specular_intensity = params["specular_intensity_bg"]
 
 # Shadeless Background
-if params["shadeless"] == "Yes":
-	material_object4.use_shadeless = True
-else:
-	material_object4.use_shadeless = False
+# TODO: Unsupported in Blender 2.8 (not sure of workaround yet)
+#if params["shadeless"] == "Yes":
+#	material_object4.use_shadeless = True
+#else:
+#	material_object4.use_shadeless = False
```

Comment:
The [workarounds](https://blender.stackexchange.com/questions/131015/where-is-shadeless-material-option-for-blender-2-8?noredirect=1&lq=1) here seem a bit... complex, though maybe they'll turn out to be pretty easy to simplify. The linked answer to [Node for setting render colour, regardless of lighting](https://blender.stackexchange.com/a/8110) seems especially useful, though it's presented in terms of Blender UI screenshots so that'd have to be translated into script code. And on the plus side, the new version will work the same with or without the GPU renderer. (The old option, "use shadeless", was a feature of the previous internal renderer, so presumably it would've either not worked or even crashed when using the GPU renderer.)

Reply:
Yeah, I'm a bit worried that a few of these *.blend files will need to be extensively redone, especially the node-based ones... which have various "glare" and "lighting" nodes + transparency rendering as the background. Still lots of stuff broken on them in Blender 2.80.

Reply:
This one actually isn't that bad after all, wrapping my head around the new interface was the trickiest part, but once I worked that out replicating the setup of Background.Material to use an Emission shader for the camera (as described in that "Node for setting render colour..." issue above was pretty straightforward. Unfortunately, I have to start over and redo it cleanly, and can't right now because it's about to rain and our roof leaks, so I have more pressing issues to deal with. :laughing:

Reply:
I think I've got it, and even managed to make it configurable by adding a second material definition with the Light Path - gated Emission shader mixed in, called "Background.Material.Shadeless", with a fake user so it stays in the Blend file even though it's unassigned. 

The Python code just assigns one or the other material, depending on the value of `use_shadeless`.

..Honestly, having done that, I can't see that it makes _any_ difference. The background appear pretty much shadeless even if that option's disabled. But, meh, now it's there. (And there is some _vaguely_-detectable difference in the color tone, at times.)

Reply:
Pushed that commit onto the branch, and a second one to save/restore keyboard focus when the preview updates. It was driving me nuts that I couldn't focus the preview frame slider and just use the arrow keys to switch frames, because every time the preview refreshed it would lose focus. So, now I can!

Reply:
Arrrgh. I just realized why the param doesn't seem to make a difference, the code doesn't actually _change_ the material assignment for the background right now. So, the param still has no effect, in terms of the visible render. (And I apparently have a vivid imagination, thinking I saw any difference at all.)  Oops.

Reply:
And now that I look at it, this wouldn't be easy at all because the color needs to be a keyframable parameter, but which parameter that is would be totally different in the node-shaded material case. I'm going to revert the changes I made to `colors.py`/`colors.blend`, since they don't work. I suspect the best option might just be to dump "use_shadeless". The animation looks fine without it.

Reply:
I just downloaded Blender 2.8 and latest (at this hour) OpenShot appimage and updated the executable path in OpenShot. However, I got the typical window with error "You need blender ....". I even closed an reopen OpenShot, but nothing changes ...